packages/expo-firebase-analytics/src/ExpoFirebaseAnalytics.web.ts

```diff
@@ -0,0 +1,65 @@
+// import * as firebase from "firebase/app";
+// // side-effect
+// import "firebase/analytics";
+
+function getFirebaseModule() {
+  try {
+    return require('firebase/app');
+  } catch ({ message }) {
+    throw new Error('Firebase JS SDK is not installed: ' + message);
+  }
+}
+function getAnalyticsModule() {
+  try {
+    const firebase = getFirebaseModule();
+    require('firebase/analytics');
+    return firebase.analytics();
+  } catch ({ message }) {
+    throw new Error('Firebase JS Analytics SDK is not available: ' + message);
+  }
+}
+
+export default {
+  get name(): string {
+    return 'ExpoFirebaseAnalytics';
+  },
+  async initializeAppDangerously(config: { [key: string]: any }): Promise<void> {
+    getFirebaseModule().initializeApp(config);
+  },
+  async deleteApp(config: { [key: string]: any }): Promise<void> {
```

Comment:
Arguments to this method are never used. Maybe it would make sense to remove it altogether?